src/project-tests/pomodoro-clock-tests.js

```diff
@@ -18,12 +18,12 @@ export default function createPomodoroClockTests() {
   }
 
   function getMinutes(str) {
-    const matches = (/^(\d{1,4})\s?([\.:,\/]\s?\d{2}.*)?$/g).exec(str);
+    const matches = (/(\d{1,2})\s?([\.:,\/]\s?\d{2}.*)?$/g).exec(str);
```

Comment:
Can you please document what this regex does? And give some example strings that would pass and some that would fail?